src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -490,33 +477,10 @@ export default class TutorialView {
 
         this.tutorial_step({
             title: 'Approach Clearances, part 2',
-            text: ['You may choose to enter one command at a time, but air traffic controllers usually do multiple. Particularly in approach clearances,',
```

Comment:
We should still mention that you can enter multiple commands together

Reply:
Yeah cool, I'll add that back in.